packages/quick_blue/lib/src/method_channel_quick_blue.dart

```diff
@@ -45,10 +45,14 @@ class MethodChannelQuickBlue extends QuickBluePlatform {
   }
 
   final Stream<dynamic> _scanResultStream = _eventScanResult.receiveBroadcastStream({'name': 'scanResult'});
+  final Stream<dynamic> _availabilityChangeStream = _eventScanResult.receiveBroadcastStream({'name': 'availabilityChange'});
 
   @override
   Stream<dynamic> get scanResultStream => _scanResultStream;
 
+  @override
+  Stream<dynamic> get availabilityChangeStream => _availabilityChangeStream;
```

Comment:
Move `get availabilityChangeStream` above `startScan` and below `_availabilityChangeStream`